ui/src/ScholarReader.tsx

```diff
@@ -404,12 +405,46 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
           undefined,
           { name: "XYZ" },
           box.left + SCROLL_OFFSET_X,
-          box.top + SCROLL_OFFSET_Y
+          box.top + SCROLL_OFFSET_Y,
         ]
       });
     }
   }
 
+  /**
+   * Will scroll a symbol horizontally into view when the drawer opens
+   * if it is now obscured by the drawer.
+   */
+  scrollSymbolHorizontallyIntoView() {
+    const { selectedSymbol, pdfViewer, pages } = this.state;
+    const DRAWER_WIDTH = 470;
+    const SYMBOL_VIEW_PADDING = 50;
+    if (pdfViewer && selectedSymbol) {
+      const symBounds = selectedSymbol.bounding_boxes[0];
+      const { x } = pdfViewer.container.getBoundingClientRect() as DOMRect;
```

Comment:
```suggestion
      const { pageLeft } = pdfViewer.container.getBoundingClientRect() as DOMRect;
```

Nit pick: just a suggestion for a more descriptive variable name.

Reply:
Unfortunately can't commit this as it's an object de-structure. But I'll rename this and recommit :)